### DISCUSSION TITLE
Automatically refresh the stream when error #2000 occurs

### DISCUSSION THREAD

**Discussion by gebopo7091** [night/betterttv#7145] [Ideas]
Can you please add an option that will automatically refresh the Twitch stream when any error occurs (#2000, #3000, #4000, etc.)?
Basically the same feature that added by the "Twitch Error AutoRefresher" extension.

**Comment by night** [comment]
Closing as a duplicate of #6756. Please add your upvote and any extra detail over there so feedback stays consolidated.
